### DISCUSSION TITLE
Getting files by language

### DISCUSSION THREAD

**Discussion by sparkles-laurel** [github-linguist/linguist#5568] [Q&A] [answered]
Is there a function in the library that implements the `---breakdown` option?

**Comment by lildude** [comment, marked as answer]
> Is there a function in the library that implements the `---breakdown` option?

[Yes](https://github.com/github/linguist/blob/bc56fd06e3b9669adfd4ae301ac2d14b68d71517/bin/github-linguist#L63), but your question doesn't match the title 😉
